wmn-data.json

```diff
@@ -5984,21 +5986,16 @@
         "cat" : "blog"
       },
       {
-        "name" : "Virustotal",
-        "uri_check" : "https://www.virustotal.com/ui/users/{account}",
+        "name" : "VirusTotal",
+        "uri_check" : "https://www.virustotal.com/ui/users/{account}/avatar",
         "pretty_uri" : "https://www.virustotal.com/gui/user/{account}",
-        "headers" : {
-                "User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:127.0) Gecko/20100101 Firefox/127.0",
-                "X-Tool": "vt-ui-main",
-                "Accept-Ianguage": "en-US,en;q=0.9,es;q=0.8",
-                "X-VT-Anti-Abuse-Header": "MTAxOTFwMDcxOTEtWkc5dWRDQmlaU0JsZG2scy5xNzE4Mjc1NDI0LjUzMw=="
-        },
         "e_code" : 200,
-        "e_string" :"data",
+        "e_string" : "",

```

Comment:
Hey, I'm a  bit late.
I made the virustotal check not so long ago, and it's still working as I've tested now.
/ui/ requires javascript rendering, but /gui/ doesn't.
/gui/ needs few specific headers to work, that's all. @3xp0rt, if you checked from your browser and not from a script adding the headers to the request, it would not work.

Reply:
Unfortunately, gui requires JavaScript. Check again.

Reply:
Yeah, I mixed up /ui/ and /gui/.
/gui/ requires js rendering. But /ui/ doesn't.

Just now (httpx):
![wmn-virustotal](https://github.com/user-attachments/assets/00a07974-fb2a-42b3-a1d3-6bc523474df5)